### ISSUE TITLE
[Feature]: Input_datetime card

### ISSUE THREAD

**Issue by Svetlinski1985** [piitaya/lovelace-mushroom#1214]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Is your feature request related to a problem?

I'm missing a input_datetime - card (similar to input_select - card) to select a certain starting/ending time for a automation. So I'm not able to create a unified dashboard.

![Screenshot_20230716_112010_Home Assistant](https://github.com/piitaya/lovelace-mushroom/assets/115838400/f35c9e30-08f6-4a65-aecd-ff8e6fbcc22f)

![Screenshot_20230716_112021_Home Assistant](https://github.com/piitaya/lovelace-mushroom/assets/115838400/2a666e53-3a0f-4961-b2a4-a329cdf2ab15)

### Describe the solution you'd like

Create a input_datetime card.

**Comment by valantur** [comment]
yeah I would really love to be able to click on + and - buttons to change the time of an input_datetime helper. Same functionality as [htmltiger/numberbox-card](https://github.com/htmltiger/numberbox-card) but with mushroom card style and customizations.

**Comment by bdsoha** [comment]
Any chance this will be implemented?

**Comment by Dimi27** [comment]
+1

**Comment by samuelecarpene** [comment]
any updates?
